### PR TITLE
feature(all): few fixes and changes

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts [hunk 1]
```diff
@@ -227,6 +218,7 @@ const PluginAttributesManager = types
               if (
                 posBusTopic === self.pluginId &&
                 posBusAttributeName === AttributeNameEnum.STATE &&
+                value &&
```
Comment (dmitry-yudakov) [inline packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:221]: You sure about this? What if it's empty string?

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts [hunk 2]
```diff
@@ -253,54 +245,50 @@ const PluginAttributesManager = types
 
         // TODO: Temporary, change to above after PosBus supports attribute items
         useStateItemRemove(key, callback) {
-          return usePosBusEvent('space-attribute-removed', (posBusTopic, posBusAttributeName) => {
-            if (posBusTopic === self.pluginId && posBusAttributeName === AttributeNameEnum.STATE) {
-              callback();
+          return usePosBusEvent(
```
Comment (PAKovalev) [inline packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:248]: why this stuff is in store? It seems it should be defined on page. not?

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: packages/plugin_google_drive/src/pages/GoogleDrivePage/GoogleDrivePage.tsx [hunk 3]
```diff
@@ -13,48 +13,48 @@ const GoogleDrivePage: FC = () => {
...
-  if (!spaceId) {
+  if (!objectId) {
     return null;
   }
 
   return (
     <SpacePage>
       <SpaceTopBar
-        title={store.spaceName ?? ''}
+        title={pluginName ?? ''}
```
Comment (dmitry-yudakov) [inline packages/plugin_google_drive/src/pages/GoogleDrivePage/GoogleDrivePage.tsx:42]: I don't think you need pluginName in order to show "Google Drive" or smt in plugin_google_drive :)

File: packages/sdk/tsconfig.json [hunk 4]
```diff
@@ -7,6 +7,6 @@
     "noEmit": false,
     "baseUrl": "src"
   },
-  "include": ["src", "../app/src/api/enums/attributeName.enum.ts"],
+  "include": ["src"],
```
Comment (dmitry-yudakov) [inline packages/sdk/tsconfig.json:10]: it was in 2 places in the original PR

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts (hunk 2 above)
Comment (Daboo1998) [reply packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:248]: Then I need to define pluginAPi put outside of the store

File: packages/sdk/tsconfig.json (hunk 4 above)
Comment (Daboo1998) [reply packages/sdk/tsconfig.json:10]: I don't see anywhere else it anymore

File: packages/plugin_google_drive/src/pages/GoogleDrivePage/GoogleDrivePage.tsx (hunk 3 above)
Comment (Daboo1998) [reply packages/plugin_google_drive/src/pages/GoogleDrivePage/GoogleDrivePage.tsx:42]: lets leave it for for now

File: packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts (hunk 1 above)
Comment (Daboo1998) [reply packages/app/src/core/models/PluginAttributesManager/PluginAttributesManager.ts:221]: Attribute value cannot be an empty string, its always an object

**Review by PAKovalev — APPROVED** [review]

**Pull request merged**